From 27c6d2c0171460e2177f48a30e95ed0ec32dd756 Mon Sep 17 00:00:00 2001
From: Georges Basile Stavracas Neto <georges.stavracas@gmail.com>
Date: Fri, 26 Feb 2021 19:17:02 -0300
Subject: [PATCH] time-zone-monitor: Switch to g_time_zone_new_identifier()

We don't even need to check for NULL here because if the system timezone
is invalid, we're too screwed to have a fallback path anyway.
---
 src/core/gcal-time-zone-monitor.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/gcal-time-zone-monitor.c b/src/core/gcal-time-zone-monitor.c
index 26325e01..35f92d1a 100644
--- a/src/core/gcal-time-zone-monitor.c
+++ b/src/core/gcal-time-zone-monitor.c
@@ -71,7 +71,7 @@ on_timedate1_proxy_properties_changed_cb (GcalTimeZoneMonitor *self,
   if (timezone_variant)
     timezone_identifier = g_variant_get_string (timezone_variant, NULL);
 
-  self->timezone = g_time_zone_new (timezone_identifier);
+  self->timezone = g_time_zone_new_identifier (timezone_identifier);
 
   g_debug ("System timezone is %s", g_time_zone_get_identifier (self->timezone));
 
-- 
GitLab

